crates/typst-library/src/math/style.rs

```diff
@@ -86,46 +84,44 @@ pub fn sans(
 /// We establish $cal(P) != scr(P)$.
 /// ```
 ///
-/// (The box is not conceptually necessary, but unfortunately currently needed
-/// due to limitations in Typst's text style handling in math.)
+/// （ボックスは概念的には不要ですが、現在のTypstの数式テキストスタイル処理の制約により必要です。）
```

Comment:
JTF日本語標準スタイルガイドに従って句点を削除して問題ないと思います。
```suggestion
/// （ボックスは概念的には不要ですが、現在のTypstの数式テキストスタイル処理の制約により必要です）
```